src/bin/cql-stress-cassandra-stress/java_generate/values/hex_blob.rs

```diff
@@ -0,0 +1,191 @@
+use super::ValueGenerator;
+use crate::java_generate::distribution::Distribution;
+
+/// Based on c-s HexBytes generator.
+/// Used by c-s to generate blob partition keys.
+/// TBH, not quite sure why c-s won't use Bytes generator for this.
```

Comment:
As I can see, `HexBlob` just generates a sequence of hexadecimal digits instead of random bytes from range 0-255.